src/test/java/uk/gov/laa/ccms/caab/service/ApplicationServiceTest.java

```diff
@@ -181,6 +193,61 @@ void createApplication_success() {
 
   }
 
+  @Test
+  void getApplicationSummary_returnsApplicationSummary_Successful() {
+    String applicationId = "12345";
+
+    // Create mock data for successful Mono results
+    RelationshipToCaseLookupDetail orgRelationshipsDetail = new RelationshipToCaseLookupDetail();
+    orgRelationshipsDetail.addContentItem(new RelationshipToCaseLookupValueDetail());
+
+    RelationshipToCaseLookupDetail personRelationshipsDetail = new RelationshipToCaseLookupDetail();
+    personRelationshipsDetail.addContentItem(new RelationshipToCaseLookupValueDetail());
+
+    AuditDetail auditDetail = new AuditDetail();
+    auditDetail.setLastSaved(Date.from(Instant.now()));
+    auditDetail.setLastSavedBy("TestUser");
+
+    CostStructure costStructure = new CostStructure();
+    costStructure.setAuditTrail(auditDetail);
+
+    Client client = new Client();
+    client.setFirstName("bob");
+    client.setSurname("ross");
+
+    StringDisplayValue applicationType = new StringDisplayValue();
+    applicationType.id("test 123");
+    applicationType.setDisplayValue("testing123");
+
+    ApplicationDetail mockApplicationDetail = new ApplicationDetail();
+    mockApplicationDetail.setAuditTrail(auditDetail);
+    mockApplicationDetail.setClient(client);
+    mockApplicationDetail.setApplicationType(applicationType);
+    mockApplicationDetail.setProceedings(new ArrayList<>());
+    mockApplicationDetail.setPriorAuthorities(new ArrayList<>());
+    mockApplicationDetail.setOpponents(new ArrayList<>());
+    mockApplicationDetail.setCosts(costStructure);
+
+    // Mock the behavior of your dependencies
+    when(ebsApiClient.getOrganisationRelationshipsToCaseValues()).thenReturn(
+        Mono.just(orgRelationshipsDetail));
+
+    when(ebsApiClient.getPersonRelationshipsToCaseValues()).thenReturn(
+        Mono.just(personRelationshipsDetail));
+
+    when(caabApiClient.getApplication(applicationId)).thenReturn(
+        Mono.just(mockApplicationDetail));
+
+    Mono<ApplicationSummaryDisplay> summaryMono =
+        applicationService.getApplicationSummary(applicationId);
+
+    // Verify the result
+    StepVerifier.create(summaryMono)
+        .expectNextMatches(summary -> summary != null)
```

Comment:
can we test for anything else in the summary based on the values at the top of the test?

Reply:
Added some asserts, but most of those checks are done in the mapper